### ISSUE TITLE
在我手机上崩溃,用readMe里提供的apk

### ISSUE THREAD

**Issue by Reginer** [Kyson/AndroidGodEye#13]
5.1的系统
```
 Process: cn.hikyson.godeyedemo, PID: 27087
    io.reactivex.exceptions.OnErrorNotImplementedException: totalTime must greater than 0
        at io.reactivex.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:704)
        at io.reactivex.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:701)
        at io.reactivex.internal.observers.LambdaObserver.onError(LambdaObserver.java:74)
        at io.reactivex.observers.SerializedObserver.onError(SerializedObserver.java:155)
        at io.reactivex.internal.operators.observable.ObservableConcatMap$SourceObserver$InnerObserver.onError(ObservableConcatMap.java:254)
        at io.reactivex.internal.observers.BasicFuseableObserver.onError(BasicFuseableObserver.java:100)
        at io.reactivex.internal.observers.BasicFuseableObserver.fail(BasicFuseableObserver.java:110)
        at io.reactivex.internal.operators.observable.ObservableMap$MapObserver.onNext(ObservableMap.java:61)
        at io.reactivex.internal.operators.observable.ObservableTimer$TimerObserver.run(ObservableTimer.java:67)
        at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
        at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26)
        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
        at java.lang.Thread.run(Thread.java:818)
     Caused by: cn.hikyson.godeye.core.internal.exception.GodEyeInvalidDataException: totalTime must greater than 0
        at cn.hikyson.godeye.core.internal.modules.cpu.CpuEngine$3.apply(CpuEngine.java:61)
        at cn.hikyson.godeye.core.internal.modules.cpu.CpuEngine$3.apply(CpuEngine.java:55)
        at io.reactivex.internal.operators.observable.ObservableMap$MapObserver.onNext(ObservableMap.java:59)
        at io.reactivex.internal.operators.observable.ObservableTimer$TimerObserver.run(ObservableTimer.java:67) 
        at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38) 
        at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26) 
        at java.util.concurrent.FutureTask.run(FutureTask.java:237) 
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152) 
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587) 
        at java.lang.Thread.run(Thread.java:818) 
```

**Comment by Luckylll** [comment]
FATAL EXCEPTION: RxComputationThreadPool-2
                                                                  Process: com.farmers_helper, PID: 7126
                                                                  io.reactivex.exceptions.OnErrorNotImplementedException: not valid ratio
                                                                      at io.reactivex.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:704)
                                                                      at io.reactivex.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:701)
                                                                      at io.reactivex.internal.observers.LambdaObserver.onError(LambdaObserver.java:74)
                                                                      at io.reactivex.observers.SerializedObserver.onError(SerializedObserver.java:155)
                                                                      at io.reactivex.internal.operators.observable.ObservableConcatMap$SourceObserver$InnerObserver.onError(ObservableConcatMap.java:255)
                                                                      at io.reactivex.internal.observers.BasicFuseableObserver.onError(BasicFuseableObserver.java:100)
                                                                      at io.reactivex.internal.observers.BasicFuseableObserver.fail(BasicFuseableObserver.java:110)
                                                                      at io.reactivex.internal.operators.observable.ObservableMap$MapObserver.onNext(ObservableMap.java:61)
                                                                      at io.reactivex.internal.operators.observable.ObservableTimer$TimerObserver.run(ObservableTimer.java:67)
                                                                      at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
                                                                      at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26)
                                                                      at java.util.concurrent.FutureTask.run(FutureTask.java:237)
                                                                      at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152)
                                                                      at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265)
                                                                      at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
                                                                      at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
                                                                      at java.lang.Thread.run(Thread.java:818)
                                                                   Caused by: cn.hikyson.godeye.core.internal.exception.GodEyeInvalidDataException: not valid ratio
                                                                      at cn.hikyson.godeye.core.internal.modules.cpu.CpuEngine$3.apply(CpuEngine.java:70)
                                                                      at cn.hikyson.godeye.core.internal.modules.cpu.CpuEngine$3.apply(CpuEngine.java:55)
                                                                      at io.reactivex.internal.operators.observable.ObservableMap$MapObserver.onNext(ObservableMap.java:59)
                                                                      at io.reactivex.internal.operators.observable.ObservableTimer$TimerObserver.run(ObservableTimer.java:67) 
                                                                      at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38) 
                                                                      at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26) 
                                                                      at java.util.concurrent.FutureTask.run(FutureTask.java:237) 
                                                                      at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152) 
                                                                      at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265) 
                                                                      at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112) 
                                                                      at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587) 
                                                                      at java.lang.Thread.run(Thread.java:818)

**Comment by Miverk** [comment]
我和你的是一样的 我的系统是4.4的

**Comment by Kyson** [comment]
最新版本1.7.4已经解决此类crash问题，谢谢大家反馈
resolved at version 1.7.4, thank you .
